tests/services/kernels/test_api.py

```diff
@@ -10,6 +10,8 @@
 from jupyter_server.utils import url_path_join
 from ...utils import expected_http_error
 
+from ipython_genutils.py3compat import PY3
+NATIVE_KERNEL_NAME = 'python3' if PY3 else 'python2'
```

Comment:
This is removed as part of the JKM changes.  NATIVE_KERNEL_NAME no longer exists.

Reply:
Thx. removed that bit.